SU2_CFD/include/solvers/CFVMFlowSolverBase.hpp

```diff
@@ -312,7 +317,7 @@ class CFVMFlowSolverBase : public CSolver {
    * \param[in] val_marker - Surface marker where the boundary condition is applied.
    */
   inline void BC_Euler_Wall(CGeometry* geometry, CSolver** solver_container, CNumerics* conv_numerics,
-                            CNumerics* visc_numerics, CConfig* config, unsigned short val_marker) final {
+                            CNumerics* visc_numerics, CConfig* config, unsigned short val_marker) override {
     /*--- Call the equivalent symmetry plane boundary condition. ---*/
     BC_Sym_Plane(geometry, solver_container, conv_numerics, visc_numerics, config, val_marker);
```

Comment:
Only one of BC_Euler_Wall / BC_Sym_Plane needs to be demoted from final to override.